bootstrap/Old_BootStrapBackup/_bootstrap_common.py

```
@@ -0,0 +1,340 @@
+        ],
+        env_overrides={"PGPASSWORD": db_conf["password"]},
+        capture_output=True,
+    )
+    return (result.stdout or "").strip()
+
+
+def ensure_database_exists(db_conf: dict[str, str]) -> None:
+    db_name = db_conf["database"]
+    exists = psql_query(
+        db_conf,
+        f"SELECT 1 FROM pg_database WHERE datname = {_sql_literal(db_name)};",
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)